examples/intrinsics.rs

```diff
@@ -264,14 +264,18 @@ mod intrinsics {
 
     /* i32 operations */
 
+    // floatsisf
+    pub fn aeabi_i2f(x: i32) -> f32 {
+        x as f32
+    }
+
     // floatsidf
     pub fn aeabi_i2d(x: i32) -> f64 {
         x as f64
     }
 
-    // floatsisf
-    pub fn aeabi_i2f(x: i32) -> f32 {
-        x as f32
+    pub fn floatsitf(x: i32) -> f128 {
```

Comment:
These need `#[cfg(f128_enable)]`, right?

Reply:
Do tests in this repo ever get run with cranelift? This "example" is what tests linking when the current crate is used as `compiler_builtins` rather than the sysroot.